style-guide-myst.md

```diff
@@ -0,0 +1,827 @@
+Use MyST syntax to automatically extract the link text. When overriding the link text, use Markdown syntax.
+
+```{list-table}
+   :header-rows: 1
+
+* - Input
+  - Output
+  - Output on GitHub
+  - Status
+* - `` {doc}`index` ``
+  - {doc}`index`
+  - {doc}<span></span>`index`
+  - Preferred.
+* - `[](index)`
+  - [](index)
+  -
+  - Do not use.
+* - `[Index page](index)`
+  - [Index page](index)
+  - [Index page](index)
+  - Preferred when overriding the link text.
+* - `` {doc}`Index page <index>` ``
+  - {doc}`Index page <index>`
+  - {doc}<span></span>`Index page <index>`
+  - Alternative when overriding the link text.
+
+```
+
+Adhere to the following conventions:
+
+- Only use the `{doc}` role when you cannot use the `{ref}` role, thus only if there is no target at the top of the file and you cannot add it. When using the `{doc}` role, your reference will break when a file is renamed or moved.
+- Override the link text only when it is necessary. If you can use the document title as link text, do so, because the text will then update automatically if the title changes.
+- Never "override" the link text with the same text that would be generated automatically.
+
+## Navigation
+
+Every documentation page must be included as a sub-page to another page in the navigation.
+
+This is achieved with the [`toctree`](https://www.sphinx-doc.org/en/master/usage/restructuredtext/directives.html#directive-toctree) directive in the parent page: <!-- wokeignore:rule=master -->
+
+````
+```{toctree}
+:hidden:
+
+sub-page1
+sub-page2
+```
+````
+
+If a page should not be included in the navigation, you can suppress the resulting build warning by putting the following instruction at the top of the file:
+
+```
+---
+orphan: true
+---
+```
+
+Use orphan pages sparingly and only if there is a clear reason for it.
+
+```{tip}
+Instead of hiding pages that you do not want to include in the documentation from the navigation, you can exclude them from being built.
+This method will also prevent them from being found through the search.
+
+To exclude pages from the build, add them to the `custom_excludes` variable in the {file}`custom_conf.py` file.
+```
+
+## Lists
+
+````{list-table}
+   :header-rows: 1
+
+* - Input
+  - Output
+* - ```
+    - Item 1
+    - Item 2
+    - Item 3
+    ```
+  - - Item 1
+    - Item 2
+    - Item 3
+* - ```
+    1. Step 1
+    1. Step 2
+    1. Step 3
+    ```
+  - 1. Step 1
+    1. Step 2
+    1. Step 3
+* - ```
+    1. Step 1
+       - Item 1
+         * Sub-item
+       - Item 2
+    1. Step 2
+       1. Sub-step 1
+       1. Sub-step 2
+    ```
+  - 1. Step 1
+       - Item 1
+         * Sub-item
+       - Item 2
+    1. Step 2
+       1. Sub-step 1
+       1. Sub-step 2
+````
+
+Adhere to the following conventions:
+
+- In numbered lists, use `1.` for all items to generate the step numbers automatically.
```

Comment:
Perhaps also mention that if the first item in the numbered list starts with a number greater than `1`, the numbering starts with that number accordingly (but this is rarely used)

Reply:
Good point, thanks! Added.